service/k8sclient/k8sclient.go

```diff
@@ -326,12 +325,35 @@ var (
 	ErrNotFound error = errors.New("resource was not found in Kubernetes cluster")
 )
 
+var pmmClientImage string
+
 // K8sClient is a client for Kubernetes.
 type K8sClient struct {
 	kubeCtl *kubectl.KubeCtl
 	l       logger.Logger
 }
 
+func init() {
+	if pmmversion.PMMVersion == "" {
+		// Prevent panicing on local development builds.
+		pmmClientImage = "perconalab/pmm-client:dev-latest"
+		return
+	}
+
+	v, err := goversion.NewVersion(pmmversion.PMMVersion)
+	if err != nil {
+		panic("failed to decide what version of pmm-client to use: " + err.Error())
+	}
+
+	if v.Core().String() == v.String() {
+		// Production version contains only major.minor.patch ...
+		pmmClientImage = "perconalab/pmm-client:2"
```

Comment:
https://hub.docker.com/r/percona/pmm-client

should be `percona/pmm-client:2`

Reply:
done

Reply:
don't see new commits so far

Reply:
Sorry, I did not noticed it failed to push. Now it's done done.